src/main/java/core/basesyntax/dataservices/DataReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.dataservices;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataReaderImpl implements DataReader {
+    @Override
+    public List<String> read(String fileName) {
+        List<String> transactions = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            reader.readLine();
```

Comment:
It looks like you are skipping the first line of the file without any condition. If the first line contains headers, it would be better to check if the line is a header or add a comment explaining why the first line is skipped.